test/src/k-provider/ovp/be-data.js

```diff
@@ -2922,7 +2922,7 @@ const EntryWithBumperWithKs = {
       playbackCaptions: [
         {
           label: 'En',
-          format: '1',
+          format: '4',
```

Comment:
is this change needed?

Reply:
Yes, I want the parser to use VTT caption in this case to check query param and we're taking the VTT for format 2 and 4.